python/sources/MQTT/main.py

```diff
@@ -32,13 +32,11 @@
 
 # setting callbacks for different events to see if it works, print the message etc.
 def on_connect(client, userdata, flags, rc, properties=None):
-    print("CONNACK received with code %s." % rc)
-
-    # handle the message in the 'connected' function
-    mqtt_functions.handle_mqtt_connected()
-
-    print("CONNECTED!")  # required for Quix to know this has connected
-
+    if rc == 0:
+        mqtt_functions.handle_mqtt_connected()
+        print("CONNECTED!") # required for Quix to know this has connected
+    else:
+        print("Connection refused. Error code: {}".format(rc))
```

Comment:
```
print("ERROR: " ....
```
I think the format for error on setup is the above, otherwise not sure if understands.

Reply:
ee0ec5e8925e340f6e5ff95d97a025be1ba1d7da